geoportailv3/static/js/measure/measuredirective.js

```diff
@@ -120,6 +154,39 @@ app.MeasureController = function($scope, ngeoDecorateInteraction) {
   ngeoDecorateInteraction(measureAzimut);
   map.addInteraction(measureAzimut);
 
+
+  goog.events.listen(measureProfile.drawInteraction, ol.DrawEventType.DRAWEND,
+      function(evt) {
+        var geom = /** @type {ol.geom.Geometry} */ (evt.feature.getGeometry());
+        var encOpt = {
+          dataProjection: 'EPSG:2169',
+          featureProjection: this['map'].getView().getProjection()
+        };
+        var req = $.param({
+          'geom': new ol.format.GeoJSON().writeGeometry(geom, encOpt),
+          'nbPoints': 100,
+          'layers': 'dhm'
+        });
+        var config = {
+          headers: {'Content-Type': 'application/x-www-form-urlencoded'}
+        };
+        this['profileOpen'] = true;
+        $http.post('profile.json', req, config).then(
```

Comment:
So I guess we should use `appModule.constant` in the `index.html` template like for the other web services. See https://github.com/Geoportail-Luxembourg/geoportailv3/blob/master/geoportailv3/templates/index.html#L188-L193.